CMakeLists.txt

```diff
@@ -97,6 +101,22 @@ include(SetRpath)
 include(DeclareMod2CSubModule)
 include(FindPkgConfig)
 include(CTest)
+include(Findnmodl)
```

Comment:
having a nice `find_package` would be cleaner. This is probably out of the scope of this PR. But definitely worth a ticket!